backend/api/tests/test_admin.py

```
@@ -4,8 +4,11 @@
 from django.urls import reverse
 from django.utils import timezone
 import datetime
```

Ruff (0.8.2)

`datetime` imported but unused

Remove unused import: `datetime`

(F401)